native/src/main/java/io/ballerina/stdlib/sql/utils/ErrorGenerator.java

```diff
@@ -95,6 +95,15 @@ public static BError getSQLApplicationError(ApplicationError error) {
                 StringUtils.fromString(message), null, null);
     }
 
+    public static BError getSQLError(Throwable th, String thMessage) {
+        String message = th.getMessage();
+        if (message == null) {
+            message = th.getClass().getName();
```

Comment:
Do we have the error thrown class name info every time when the message is not null ?

Reply:
No, The assumption is if there is an error message the user has the info already without knowing about the class name.